SoundSwitch/Framework/Configuration/SoundSwitchConfiguration.cs

```diff
@@ -36,7 +38,7 @@ public SoundSwitchConfiguration()
         {
             // Basic Settings
             FirstRun = true;
-            SwitchForegroundProgram = true;
+            SwitchForegroundProgram = false;
```

Comment:
mmmmh, I thought it could conflict with the profile feature in some way.

This feature will by default reset all Windows 10 audio settings to avoid having some weirdness where you switched the audio device, but your previous foreground app don't change.

This then conflict with the profile feature. I'm thinking of disabling this one completely. The user should choose between the two. But again, not simple to explain it when you don't have a good understanding of the way it works in Windows 10 ...

Hence, for now, disabled by default. I rather have the user set up a profile than use this.